modules/ROOT/pages/mule-application-about.adoc

```diff
@@ -24,6 +24,8 @@ For example, the Choice router is a component that dynamically routes messages t
 .A list of the core components in Anypoint Studio
```

Comment:
Can we *not* use "components" generically?

Reply:
I didn't write this doc. I'm clarifying the terminology.